Gemfile

```diff
@@ -59,7 +59,7 @@ gem "pg", ">= 0.18", "< 2.0"
 gem "phonelib"
 gem "pry-rails"
 gem "rack-attack"
-gem "rack-mini-profiler"
+gem "rack-mini-profiler", require: false
```

Comment:
We don't want this to autoload anymore, as it clutters up Datadog w/ its POST requests.